### ISSUE TITLE
Add a setting for the maximum number of posts in one status

### ISSUE THREAD

**Issue by stevejburge** [publishpress/PublishPress-Planner#1725]
This is a potentially useful customer request. 

They would like to set a maximum number of posts in one status on the Content Calendar. They have many articles in a single status, as shown in this screenshot.

Could we allow a maximum of 10 posts (for example), then show an accordion to uncover more?

![image002](https://github.com/user-attachments/assets/d99b9c04-4dc0-4fad-8810-0d5f8bf679f4)

**Comment by olatechpro** [comment]
@stevejburge I'm wondering if something is broken in the version you're testing because long posts shouldn't show by default without a show more and hide more button?

![show](https://github.com/user-attachments/assets/67f24066-d509-41bd-8551-26cbf14034df)

**Comment by stevejburge** [comment]
@olatechpro Good point. I can debug that with them. I guess this setting remains a separate improvement idea
